examples/saml-auto-reg/pom.xml

```diff
@@ -0,0 +1,73 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wildfly.wildfly-s2i.examples</groupId>
+    <artifactId>saml</artifactId>
+    <version>1.0</version>
+    <packaging>war</packaging>
+
+    <name>WildFly S2I - SAML example</name>
+    <description>An application secured with SAML example</description>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <version.maven.war.plugin>3.3.2</version.maven.war.plugin>
+        <version.wildfly>28.0.1.Final</version.wildfly>
+        <version.wildfly.plugin>4.2.0.Final</version.wildfly.plugin>
+        <version.wildfly.cloud.galleon.pack>4.0.1.Final</version.wildfly.cloud.galleon.pack>
```

Comment:
@jfdenise can we aupdate to `5.0.0.Beta2` or higher?, IIUC this is the version that foxes the issue with the EJB layer...